diff --git a/fceumm-code/src/drivers/libretro/libretro.c b/fceumm-code/src/drivers/libretro/libretro.c
index 8504acb..9d4d1c1 100644
--- a/fceumm-code/src/drivers/libretro/libretro.c
+++ b/fceumm-code/src/drivers/libretro/libretro.c
@@ -430,7 +430,7 @@ void retro_get_system_info(struct retro_system_info *info)
    info->need_fullpath = true;
    info->valid_extensions = "fds|nes|unif";
    info->library_version = "98.13mm (SVN)";
-   info->library_name = "FCEUmm";
+   info->library_name = "FCEUnext";
    info->block_extract = false;
 }
 
