--- buildroot/package/boost/boost.mk	2020-03-15 17:33:24.084387009 +0100
+++ custom/package/boost/boost.mk	2020-03-15 17:35:34.893626139 +0100
@@ -48,7 +48,7 @@
 BOOST_WITHOUT_FLAGS += $(if $(BR2_PACKAGE_BOOST_TYPE_ERASURE),,type_erasure)
 BOOST_WITHOUT_FLAGS += $(if $(BR2_PACKAGE_BOOST_WAVE),,wave)
 
-BOOST_TARGET_CXXFLAGS = $(TARGET_CXXFLAGS)
+BOOST_TARGET_CXXFLAGS = $(TARGET_CXXFLAGS) -std=c++11
 
 BOOST_FLAGS = --with-toolset=gcc
 
